test/test_tensordict.py

```diff
@@ -764,6 +764,56 @@ def test_set_nontensor(self, td_name):
         td.set("numpy", r.numpy())
         torch.testing.assert_allclose(td.get("numpy"), r)
 
+    def test_getitem_ellipsis(self, td_name):
+        torch.manual_seed(1)
+        td = TensorDict(
+            {"a": torch.randn(1, 3, 4, 5, 6), "b": torch.randn(1, 3, 4, 5)},
+            batch_size=[1, 3, 4],
+        )
+
+        actual_idx = [..., (..., 0), (0, ...), (0, ..., 0)]
+        expected_idx = [
+            (slice(None), slice(None), slice(None)),
+            (slice(None), slice(None), 0),
+            (0, slice(None), slice(None)),
+            (0, slice(None), 0),
+        ]
+
+        for i in range(len(actual_idx)):
+            actual_td = td[actual_idx[i]]
+            expected_td = td[expected_idx[i]]
+
+            for key in td.keys():
+                actual_tensor = actual_td[key]
+                expected_tensor = expected_td[key]
+
+                assert actual_tensor.shape == expected_tensor.shape
+                torch.testing.assert_allclose(actual_tensor, expected_tensor)
+
+    def test_setitem_ellipsis(self, td_name):
+        torch.manual_seed(1)
+        td = TensorDict(
+            {"a": torch.randn(1, 3, 4, 5, 6), "b": torch.randn(1, 3, 4, 5)},
+            batch_size=[1, 3, 4],
+        )
+
+        actual_idx = [..., (..., 0), (0, ...), (0, ..., 0)]
+        expected_idx = [
+            (slice(None), slice(None), slice(None)),
+            (slice(None), slice(None), 0),
+            (0, slice(None), slice(None)),
+            (0, slice(None), 0),
+        ]
+
+        for i in range(len(actual_idx)):
+            idx = actual_idx[i]
+            td_clone = td.clone()
+            actual_td = td[idx].clone().zero_()
+            td_clone[idx] = actual_td
+
+            for key in td.keys():
+                assert (td[idx].get(key) == 0).all()
+
```

Comment:
Do you think we could write unit tests for `convert_ellipsis_to_idx`?
Those would include: `...`, `(..., 0, ...)`, `(0, ..., 0)`, `(..., 0)`, `(0, ...)`, `(1:2, ...)` `(0, ..., 0, ...)` and maybe no ellipsis at all `(1, 2)` or anything you could think of :)

Reply:
Added units tests